### ISSUE TITLE
[q] Any plans to support profiling parallel programs?

### ISSUE THREAD

**Issue by sphynx** [mpickering/eventlog2html#102]
It would be nice if `eventlog2html` can produce visualizations similar to [ThreadScope](https://wiki.haskell.org/ThreadScope). That would be great, because ThreadScope is a pain to install or build, because of the GTK. 

I mean something similar to [this](https://wiki.haskell.org/ThreadScope_Tour/Profile)

**Comment by mpickering** [comment]
Indeed, and I would accept a patch which implemented some simple visualisations but seeing as I am not a user of parallel features or ThreadScope I'm not sure I can implement it myself currently.
